service.sh

```
@@ -277,4 +282,4 @@ settings put global activity_manager_constants max_cached_processes
 su -c "stop logd"
 
 # SuperMario-Tweaker Script
-"${MODDIR}/SuperMario-Tweaker.sh" > /dev/null
\ No newline at end of file
+"${MODDIR}/SuperMario-Tweaker.sh" > /dev/null
```

Shellcheck (0.10.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)